tests/docker/install/docker-compose.yml

```diff
@@ -255,5 +255,51 @@ services:
     hostname: "py3.11-alpine3.17"
     image: "python:3.11-alpine3.17"
     << : [*command-install-test, *workdir, *volumes-codebase]
+    environment:
+      << : *envvars-install-test
+
+  #--------------------------------------------------------------------
+  # Python 3.12
+  #--------------------------------------------------------------------
+
+  py3.12-install-debian11:
+    hostname: "py3.12-debian11"
+    image: "python:3.12-bullseye"
+    << : [*command-install-test, *workdir, *volumes-codebase]
+    environment:
+      << : *envvars-install-test
+
+  py3.12-install-debian12:
+    hostname: "py3.12-debian12"
+    image: "python:3.12-bookworm"
```

Comment:
I'm using Debian 11, 12 and Alpine 19, 20 because those are what's available for Python 3.12 https://hub.docker.com/_/python